agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -1518,7 +1518,7 @@ bool main_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
             LOG(DEBUG) << "slave is backhaul manager, updating";
             m_pBackhaulManagerSlave = sd;
 
-#if !defined(BEEROCKS_LINUX) && !defined(BEEROCKS_TURRIS_OMNIA)
+#ifndef BEEROCKS_TURRIS_OMNIA
```

Comment:
Whole commit is wrong. Doesn't say why it disables arp monitor on Turris Omnia, and also it Re-enables arp monitor for Linux.
What was wrong with how it used to be? Looks like this commit should be completely removed.

Reply:
Turris Omnia currently uses the same dummy BPL and the Linux platform, so the ARP monitor should be disabled for both. Later on we should create a Linux BPL with ARP monitor support and use it for all Linux based platforms.

Reply:
But now arp monitor will be enabled for Linux, is this what you meant to happen?

Does the arp monitor use the BPL? Is this why it has to be disabled when dummy BPL is used?

Reply:
No, enabling the ARP monitor for Linux is a bug.
It should be disabled for both Linux and Turris Omnia platforms.

Reply:
Still not fixed.

Reply:
removing the commit